administrator/components/com_joomgallery/helpers/adminconfig.php

```diff
@@ -96,6 +96,9 @@ public function save($newconfig = null, $id = false, $existing_row = 0, $group_i
       return false;
     }
 
+    JPluginHelper::importPlugin('content');
+    $this->_mainframe->triggerEvent('onContentBeforeSave', array(_JOOM_OPTION.'.config', &$config, $isNew, $data));
```

Comment:
$this->_mainframe is not declared in this content, this call will throw an error